### DISCUSSION TITLE
How to keep original file name while exporting?

### DISCUSSION THREAD

**Discussion by netcorefan1** [mifi/lossless-cut#1684] [Q&A] [answered]
Hello,
on all exports LLC adds "-cut-merged-xxxxxxxxxxxxx" to the file name and I have to manually rename each time. How to keep the original file name without any stuff added? I didn't saw anything in the options.
Thanks

**Comment by mifi** [comment]
Hi. see #938

**Reply by netcorefan1** [reply]
Thanks. output directory is different from input directory, so there is no need to overwrite. There is no input field as mentioned [here](https://github.com/mifi/lossless-cut/issues/96#issuecomment-766185083). I must miss something obvious.

**Reply by mifi** [reply, marked as answer]
that input field is for the separate exported segments/files. What you are trying to do is to cut&merge segments to one file. It's not yet possible to customise that file name
